### ISSUE TITLE
[Windows Terminal>Setting>Startup]: Screen reader announces updated row and column count twice when resizing rows or columns in “Launch” grid.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#19692]
### Windows Terminal version

1.25.3571.0

### Windows build number

Windows 11 Version 26H1 (OS Build 28000.1340)

### Steps to reproduce

**Repro Steps:**
1.Launch the Window terminal app.
2.Navigate to the 'New Open Tab' and select the 'Settings' menu item.
3.Settings page will open.
4.Now navigate to 'Startup' and select 'it'.
5.Startup page will open.
6.Now navigate to 'Launch size' group and expand it.
7.Now turn on the narrator using 'CTRL+Win+Enter' keys.
8.Now navigate to the 'Column edit' and change the value by up and down arrow keys.
9.Observe the issue here.

**User Experience:**
Redundant announcements add to cognitive load, making it more challenging for screen reader users to determine if multiple actions have taken place. This disrupts the expectation for concise and meaningful feedback.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachements:**
https://microsoftapc-my.sharepoint.com/:u:/g/personal/v-heparihar_microsoft_com/IQAes9kV___TSoiF8S077d3FAbbhQUV5ydCKXTgqiHDCalM?e=bAaEib

### Expected Behavior

Screen reader should announce the updated row and column information only once per resize action.

### Actual Behavior

When the user resizes rows or columns in the “Launch” grid, the screen reader announces the updated row and column information twice for a single resize action. This results in redundant feedback and creates confusion for screen reader users.
Example: If we change the size 118 then screen reader announce '118-118' twice.

**Comment by carlos-zamora** [comment]
Repros on the stable channel. Does not repro in standard NumberBox control in WinUI 2 Gallery.
